libraries/AC_AttitudeControl/AC_AttitudeControl_Heli.cpp

```diff
@@ -91,7 +91,7 @@ const AP_Param::GroupInfo AC_AttitudeControl_Heli::var_info[] = {
 
     // @Param: RAT_PIT_P
     // @DisplayName: Pitch axis rate controller P gain
-    // @Description: Pitch axis rate controller P gain.  Converts the difference between desired pitch rate and actual pitch rate into a motor speed output
+    // @Description: Pitch axis rate controller P gain.  Corrects in proportion to the difference between the desired pitch rate vs actual pitch ra
```

Comment:
Corrects in proportion to the difference between the desired pitch rate vs actual pitch

Reply:
fixed